shadowsocks-csharp/Data/ja.txt

```diff
@@ -148,3 +148,6 @@ Proxy handshake failed=プロキシ ハンドシェイクに失敗しました
 Register hotkey failed=ホットキーの登錄に失敗しました。
 Cannot parse hotkey: {0}=ホットキーを解析できません: {0}
 Timeout is invalid, it should not exceed {0}=タイムアウト値が無効です。{0} 以下の値を指定して下さい。
+
+Error occured when process proxy setting, do you want reset current setting and retry?=处理代理设置时发生错误，是否重置当前代理设置并重试？

```

Comment:
Considering there is neither suitable Japanese nor Traditional Chinese translation, you may could remove these lines and just let the UI to display its original English text.

Reply:
I'll ask my friend for a translate, will update (or delete) it.